pynecone/utils.py

```diff
@@ -893,9 +893,13 @@ def format_route(route: str) -> str:
     Returns:
         The formatted route.
     """
-    route = route.strip(os.path.sep)
+    # If the route is empty, return the index route.
+    if route == "":
+        return constants.INDEX_ROUTE
+
+    route = route.strip("/")
```

Comment:
Apparently os.path.sep was tripping it up on Windows, had to change it to an explicit "/"